src/state.js

```diff
@@ -32,6 +33,16 @@ webrtc.on('chat', msg => {
   state.chat.push(msg)
 })
 
+webrtc.on('point_out', pointsOut => {
+  if(pointsOut.state){
+    state.pointOuts.push(pointsOut.sender)
```

Comment:
wenn die property state truthy ist, kann man den sender verwenden? wie kommt das?

Reply:
Wenn er aufzeigt (true) kann der Sender in die Liste. Wenn er das aufzeigen zurück nimmt (false) wird er aus der Liste entfernt

Reply:
Gut, macht sinn. Schreibt das am besten als Kommentar dazu, was state in dem fall aussagt